[2022-07-22 12:59:50,745] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: tutorial.t2 scheduled__2022-06-05T00:00:00+00:00 [queued]>
[2022-07-22 12:59:50,939] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: tutorial.t2 scheduled__2022-06-05T00:00:00+00:00 [queued]>
[2022-07-22 12:59:50,940] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-07-22 12:59:50,942] {taskinstance.py:1377} INFO - Starting attempt 1 of 2
[2022-07-22 12:59:50,944] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-07-22 12:59:51,110] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): t2> on 2022-06-05 00:00:00+00:00
[2022-07-22 12:59:51,124] {standard_task_runner.py:52} INFO - Started process 755 to run task
[2022-07-22 12:59:51,164] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'tutorial', 't2', 'scheduled__2022-06-05T00:00:00+00:00', '--job-id', '28', '--raw', '--subdir', 'DAGS_FOLDER/nickdag.py', '--cfg-path', '/tmp/tmp51sfounr', '--error-file', '/tmp/tmpjkw7jjmj']
[2022-07-22 12:59:51,173] {standard_task_runner.py:80} INFO - Job 28: Subtask t2
[2022-07-22 12:59:51,252] {logging_mixin.py:115} WARNING - /home/***/.local/lib/python3.7/site-packages/***/configuration.py:528 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2022-07-22 12:59:52,025] {task_command.py:371} INFO - Running <TaskInstance: tutorial.t2 scheduled__2022-06-05T00:00:00+00:00 [running]> on host 2f63307804d5
[2022-07-22 12:59:52,908] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=caroline.kosalka@gmail.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=tutorial
AIRFLOW_CTX_TASK_ID=t2
AIRFLOW_CTX_EXECUTION_DATE=2022-06-05T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-06-05T00:00:00+00:00
[2022-07-22 12:59:52,917] {python.py:173} INFO - Done. Returned value was: goodbye
[2022-07-22 12:59:53,765] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=tutorial, task_id=t2, execution_date=20220605T000000, start_date=20220722T125950, end_date=20220722T125953
[2022-07-22 12:59:54,994] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-07-22 12:59:56,522] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-07-25 09:17:55,010] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: tutorial.t2 scheduled__2022-06-05T00:00:00+00:00 [queued]>
[2022-07-25 09:17:55,147] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: tutorial.t2 scheduled__2022-06-05T00:00:00+00:00 [queued]>
[2022-07-25 09:17:55,155] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-07-25 09:17:55,169] {taskinstance.py:1377} INFO - Starting attempt 1 of 2
[2022-07-25 09:17:55,172] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-07-25 09:17:55,382] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): t2> on 2022-06-05 00:00:00+00:00
[2022-07-25 09:17:55,409] {standard_task_runner.py:52} INFO - Started process 701 to run task
[2022-07-25 09:17:55,464] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'tutorial', 't2', 'scheduled__2022-06-05T00:00:00+00:00', '--job-id', '19', '--raw', '--subdir', 'DAGS_FOLDER/nickdag.py', '--cfg-path', '/tmp/tmpafn0h3di', '--error-file', '/tmp/tmpl5jvj9_a']
[2022-07-25 09:17:55,480] {standard_task_runner.py:80} INFO - Job 19: Subtask t2
[2022-07-25 09:17:55,676] {logging_mixin.py:115} WARNING - /home/***/.local/lib/python3.7/site-packages/***/configuration.py:528 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2022-07-25 09:17:56,122] {task_command.py:371} INFO - Running <TaskInstance: tutorial.t2 scheduled__2022-06-05T00:00:00+00:00 [running]> on host 01e6b1df1873
[2022-07-25 09:17:56,457] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=caroline.kosalka@gmail.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=tutorial
AIRFLOW_CTX_TASK_ID=t2
AIRFLOW_CTX_EXECUTION_DATE=2022-06-05T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-06-05T00:00:00+00:00
[2022-07-25 09:17:56,462] {python.py:173} INFO - Done. Returned value was: goodbye
[2022-07-25 09:17:56,664] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=tutorial, task_id=t2, execution_date=20220605T000000, start_date=20220725T091755, end_date=20220725T091756
[2022-07-25 09:17:56,817] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-07-25 09:17:57,284] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
